src/librustdoc/test.rs

```diff
@@ -259,8 +260,9 @@ fn run_test(test: &str, cratename: &str, filename: &str, cfgs: Vec<String>, libs
         control.after_analysis.stop = Compilation::Stop;
     }
 
+    let mir_passes = Rc::new(Passes::new());
```

Comment:
Hopefully it doesn't matter, but it seems ungreat to be supplying empty passes here, instead of some kind of default set. (a) rustdoc may eventually want to do some mir-based queries, who knows, or (b) the compiler may use mir-based queries internally and (c) anyway this probably serves as a kind of template.